src/modules/flow/oic/Makefile

```diff
@@ -1,3 +1,3 @@
 obj-$(FLOW_NODE_TYPE_OIC) += oic.mod
-obj-oic-$(FLOW_NODE_TYPE_OIC) := oic.json oic.o
+obj-oic-$(FLOW_NODE_TYPE_OIC) := core.light.json oic.json oic.o
```

Comment:
I guess the deletion of core.light.json is a mistake, but even without that, I don't like adding it here. The obj-*-y variable refers to things that will be generated. The flow modules have the .json there because that's the resulting json with the node types descriptions after passing through the node-gen script. It's not the source .json